.github/workflows/build.yml

```diff
@@ -138,6 +138,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 10
     strategy:
+      fail-fast: false
```

Comment:
I never liked the fail-fast strategy here because it made it really hard to see which e2e tests failed after a change. Therefore, I decided to disable it